library/ecjpake.c

```diff
@@ -105,6 +111,10 @@ int mbedtls_ecjpake_setup( mbedtls_ecjpake_context *ctx,
                            size_t len )
 {
     int ret;
+    ECJPAKE_VALIDATE_RET( ctx != NULL );
```

Comment:
Not a blocker: please leave a blank line after the block of declarations.

Reply:
Fixing.